src/utils/findMatchedProjectConfig.ts

```diff
@@ -0,0 +1,53 @@
+import { minimatch } from 'minimatch';
+
+import type { ProjectConfig } from '@/types';
+import { getChangedFileListFromCommit } from '@/utils/github/commit/getChangedFileListFromCommit';
+import { getDeployInformationFromContext } from '@/utils/github/deployment/getDeployInformationFromContext';
+import { logger } from '@/utils/github/logger';
+
+interface FindMatchedProjectConfigProps {
+  parseProjectConfig: ProjectConfig[];
```

Comment:
요 인자명만 바뀌면 좋겠어요.
parse는 동사형이기에 함수를 암시하게되어 어색한 부분이 있습니다.
```suggestion
  parsedProjectConfig: ProjectConfig[];
```